admin/acceptance.py

```diff
@@ -195,6 +203,23 @@ def ensure_keys(reactor):
             found.  Otherwise, fails with ``AgentNotFound`` or ``KeyNotFound``.
         """
 
+    def extend_cluster(reactor, cluster, count, tag, starting_index):
+        """
+        Create new nodes and add them to the given cluster.
+
+        :param reactor: Reactor to use.
```

Comment:
To be honest, the practice of specifying types in docstrings is not something I've seen before but we should at least be consistent in how it's done. Should then be 'Reactor reactor'. Really Reactor to use is not very descriptive - what is the reactor used for?